[CI] ClientYamlTestSuiteIT test {yaml=nodes.stats/11_indices_metrics/Metric - http} failing

**Build Scans:**
- [elasticsearch-intake #39151 / part1](https://gradle-enterprise.elastic.co/s/ha7xvfh6bbj4s)

**Reproduction Line:**
```
./gradlew ":rest-api-spec:yamlRestTest" --tests "org.elasticsearch.test.rest.ClientYamlTestSuiteIT" -Dtests.method="test {yaml=nodes.stats/11_indices_metrics/Metric - http}" -Dtests.seed=E9790FCF181605F1 -Dtests.locale=syr-IQ -Dtests.timezone=Asia/Calcutta -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.test.rest.ClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dnodes.stats%2F11_indices_metrics%2FMetric%20-%20http%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: field [nodes.$node_id.http.clients.0.agent] was expected to be of type String but is an instanceof [null]
Expected: an instance of java.lang.String
     but: null
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {yaml=nodes.stats/11_indices_metrics/Metric - http}
- [main] 1 consecutive failures in step part1
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -485,9 +485,6 @@ tests:
 - class: org.elasticsearch.index.mapper.DateRangeDocValuesLoaderTests
   method: testEmptyRangesAppendsNull
   issue: https://github.com/elastic/elasticsearch/issues/148378
-- class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
-  method: test {yaml=nodes.stats/11_indices_metrics/Metric - http}
-  issue: https://github.com/elastic/elasticsearch/issues/148430
 - class: org.elasticsearch.xpack.security.QueryableReservedRolesIT
   method: testRestartForConfiguringReservedRolesAndClosingIndex
   issue: https://github.com/elastic/elasticsearch/issues/148431

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/nodes.stats/11_indices_metrics.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/nodes.stats/11_indices_metrics.yml
@@ -314,7 +314,6 @@
   - gte: { nodes.$node_id.http.total_opened: 1 }
   - is_true:  nodes.$node_id.http.clients
   - gte: { nodes.$node_id.http.clients.0.id: 1 }
-  - match: { nodes.$node_id.http.clients.0.agent: "/.*/" }
   - match: { nodes.$node_id.http.clients.0.local_address: "/.*/" }
   - match: { nodes.$node_id.http.clients.0.remote_address: "/.*/" }
   - is_true:  nodes.$node_id.http.clients.0.last_uri
@@ -322,8 +321,7 @@
   - gte: { nodes.$node_id.http.clients.0.last_request_time_millis: 1614028911317 }
   - gte: { nodes.$node_id.http.clients.0.request_count: 1 }
   - gte: { nodes.$node_id.http.clients.0.request_size_bytes: 0 }
-  # values for clients.0.closed_time_millis, clients.0.x_forwarded_for, and clients.0.x_opaque_id are often
-  # null and cannot be tested here
+  # Values for clients.0.{agent, closed_time_millis, x_forwarded_for, x_opaque_id} can be null and cannot be tested here
 
 ---
 "Metric - blank for indices shard_stats":

